### PR TITLE
New package: MultiMC-0.4.7

### PR DESCRIPTION
**Pull request by Spencer-H**

### CODE REVIEW

File: srcpkgs/MultiMC/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+maintainer="Spencer H <spencernh77@gmail.com>"
+license="Apache 2.0"
+homepage="http://multimc.org"
+distfiles="https://github.com/MultiMC/MultiMC5/archive/0.4.7.tar.gz"
+checksum=961ece29094931b0638244c4a2614623d6df3193ee1a9e156ed36c01325ebae0
+wrksrc=${pkgname}5-$version
+
+pre_configure() {
+export JAVA_HOME=/usr/lib/jvm/openjdk
+export PATH=$PATH:$JAVA_HOME/bin
```
Comment (Gottox) [inline srcpkgs/MultiMC/template]: indent

**Comment by Spencer-H** [comment]
Fixed.

**Comment by xtraeme** [comment]
why `/opt`? please use `/usr/share/$pkgname` instead.

**Comment by Spencer-H** [comment]
Actually thanks, that just reminded me that I forgot something when I
uploaded this. The executable has to be in a folder that the user has write
access to. Would /usr/share be the right place for that, or is there
something else I should use.

**Comment by dominikh** [comment]
Considering our discussion on IRC, please ping when this PR is ready for review again.

(Summary of the discussion: write-access for all users to where MultiMC is installed isn't acceptable, MultiMC should store its files in the user's home directory. I have no idea if the current version of the template implements that)

**Comment by Spencer-H** [comment]
@dominikh This version actually does have that. I should have mentioned that when I updated the pull request. But this version moves MultiMC and all of the files it uses to /usr/share/multimc and it makes sure that the program puts anything it needs to write to into a .multimc folder in the home of the user executing it. After our discussion I went through the repo for MultiMC looking for where I could change it to treat a different directory as home. It turns out there is a flag for it you can pass to the program, which I hadn't discovered before. This version makes use of that in the script that it places into /usr/bin for executing the program.

**Comment by Spencer-H** [comment]
Added a patch that forces translations to respect the directory selected for install files. I'm going to make a pull request on the MultiMC repo for this.

**Comment by pullmoll** [comment]
You can remove the repeated `vmkdir /usr/share/multimc`.
The line `vmkdir usr/share/multimc/jars` will already create the parent.
The issues with downloading from https should be fixed now.

**Comment by Spencer-H** [comment]
This is up to head and has been ready to be merged for a while. The patch fixes it so that MultiMC runs without needing write access to anything besides the users home ( .multimc ). The patch will also be merged in the next release of MultiMC so that it does not need to be maintained.

File: srcpkgs/MultiMC/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+pkgname=MultiMC
+version=0.4.7
+revision=1
+build_style=cmake
+hostmakedepends="openjdk openjdk-jre cmake qt5-devel qt5-x11extras-devel gtk+-devel xxd"
+short_desc="An instanced Minecraft Client"
+maintainer="Spencer H <spencernh77@gmail.com>"
+license="Apache 2.0"
+homepage="http://multimc.org"
+distfiles="https://github.com/MultiMC/MultiMC5/archive/0.4.7.tar.gz"
```
Comment (dominikh) [inline srcpkgs/MultiMC/template]: Please use ${version} in the distfiles URL

File: srcpkgs/MultiMC/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'MultiMC'
+pkgname=MultiMC
+version=0.4.7
+revision=1
+build_style=cmake
+hostmakedepends="openjdk openjdk-jre cmake qt5-devel qt5-x11extras-devel gtk+-devel xxd"
```
Comment (dominikh) [inline srcpkgs/MultiMC/template]: The -devel packages should probably be makedepends, not hostmakedepends. That is, please fix the cross-compile failures for ARM that travis is reporting. (You can ignore the musl errors.)

**Comment by Spencer-H** [comment]
And fixed.

**Comment by dominikh** [comment]
I'll make the final fixes myself before merging it.

**Comment by dominikh** [comment]
Merged as 3a967c25cb26473c1d8177ef45e9bc5e910fd420

**Pull request closed without merging**